tasks/src/reinforcement-learning/about.md

```diff
@@ -54,3 +54,5 @@ Would you like to learn more about the topic? Awesome! Here you can find some cu
 - [Train a Deep Reinforcement Learning lander agent to land correctly on the Moon 🌕 using Stable-Baselines3](https://github.com/huggingface/deep-rl-class/blob/main/unit1/unit1.ipynb)
 - [Introduction to Unity MLAgents](https://colab.research.google.com/github/huggingface/deep-rl-class/blob/main/unit4/unit4.ipynb)
 - [Training Decision Transformers with 🤗 transformers](https://github.com/huggingface/blog/blob/main/notebooks/101_train-decision-transformers.ipynb)
+

```

Comment:
Maybe let's remove the incomplete section

```
Task Variants
This place can be filled with variants of this task if there's any.
```

Reply:
Reason why we have these is to prompt visitors to actually contribute. Same applies for your other comments.

Reply:
Maybe let's put a link to GitHub in the incomplete sections so when someone visits the live page, they go directly to where they can contribute